<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderDialog</class>
 <widget class="QDialog" name="OrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>订单管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="productTableWidget">
     <property name="minimumHeight">
      <number>250</number>
     </property>
     <column>
      <property name="text">
       <string>成品型号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>描述</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>BOM名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="productButtonLayout">
     <item>
      <widget class="QPushButton" name="addProductButton">
       <property name="text">
        <string>添加成品</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeProductButton">
       <property name="text">
        <string>删除成品</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="orderNumberLayout">
     <item>
      <widget class="QLabel" name="orderNumberLabel">
       <property name="text">
        <string>订单编号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="orderNumberLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="customerNameLayout">
     <item>
      <widget class="QLabel" name="customerNameLabel">
       <property name="text">
        <string>客户名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="customerNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="statusLayout">
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>状态：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="statusComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="deliveryDateLayout">
     <item>
      <widget class="QLabel" name="deliveryDateLabel">
       <property name="text">
        <string>货期：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="deliveryDateLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="orderManagerLayout">
     <item>
      <widget class="QLabel" name="orderManagerLabel">
       <property name="text">
        <string>订单负责人：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="orderManagerLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="maintainerLayout">
     <item>
      <widget class="QLabel" name="maintainerLabel">
       <property name="text">
        <string>维护人：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="maintainerLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="noteLayout">
     <item>
      <widget class="QLabel" name="noteLabel">
       <property name="text">
        <string>备注：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="remarksTextEdit">
       <property name="maximumHeight">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="checkInventoryButton">
       <property name="text">
        <string>检查库存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
